tests/e2e/performance/test_small_file_workload.py

```diff
@@ -40,31 +42,66 @@ class TestSmallFileWorkload(E2ETest):
     workloads
     """
 
+    def analize_results(self, results, logs):
+
+        def add_value(res, key, value):
+            if key not in res.keys():
+                res[key] = [value]
+            else:
+                res[key].append(value)
+            return res
+        op = ''
+        for line in logs.split('\n'):
+            log.debug(line)
+            if "operation :" in line:
+                op = line.split()[-1].strip()
+            if "total threads" in line:
+                results['smallfile-res'][op] = add_value(
+                    results['smallfile-res'][op],
+                    'total_threads', line.split()[-1].strip())
+            if "total files" in line:
+                results['smallfile-res'][op] = add_value(
+                    results['smallfile-res'][op],
+                    'total_files', line.split()[-1].strip())
+            if "elapsed time" in line:
+                results['smallfile-res'][op] = add_value(
+                    results['smallfile-res'][op],
+                    'elapsed_time', line.split()[-1].strip())
+            if "files/sec" in line:
+                results['smallfile-res'][op] = add_value(
+                    results['smallfile-res'][op],
+                    'filesPsec', line.split()[-1].strip())
+            if "requested files" in line:
+                results['smallfile-res'][op] = add_value(
+                    results['smallfile-res'][op],
+                    'percenrage', line.split()[0].strip())
+        return results
+
     @pytest.mark.parametrize(
         argnames=["file_size", "files", "threads", "samples", "interface"],
         argvalues=[
             pytest.param(
-                *[4, 50000, 4, 1, constants.CEPHBLOCKPOOL],
+                *[4, 50000, 4, 3, constants.CEPHBLOCKPOOL],
                 marks=pytest.mark.polarion_id("OCS-1295"),
             ),
             pytest.param(
-                *[16, 50000, 4, 1, constants.CEPHBLOCKPOOL],
+                *[16, 50000, 4, 3, constants.CEPHBLOCKPOOL],
                 marks=pytest.mark.polarion_id("OCS-2020"),
             ),
             pytest.param(
-                *[16, 200000, 4, 1, constants.CEPHBLOCKPOOL],
+                *[16, 200000, 4, 3, constants.CEPHBLOCKPOOL],
                 marks=pytest.mark.polarion_id("OCS-2021"),
             ),
             pytest.param(
-                *[4, 50000, 4, 1, constants.CEPHFILESYSTEM],
+                *[4, 50000, 4, 3, constants.CEPHFILESYSTEM],
                 marks=pytest.mark.polarion_id("OCS-2022"),
             ),
             pytest.param(
-                *[16, 50000, 4, 1, constants.CEPHFILESYSTEM],
+                *[16, 50000, 4, 3, constants.CEPHFILESYSTEM],
                 marks=pytest.mark.polarion_id("OCS-2023"),
             ),
             pytest.param(
-                *[16, 200000, 4, 1, constants.CEPHFILESYSTEM],
+                *[16, 200000, 4, 3, constants.CEPHFILESYSTEM],
```

Comment:
We want to remove this test. So, let's ignore updating this param.

Reply:
i delete this test.